docs/marketplace/dev/architecture-overview/202108.0/marketplace-architecture-overview-application-composition.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Marketplace Application Composition
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace looks like in general. After reading this odc you will now: 
+
+- how the Spryker Marketplace is composed on application level
+
+
+## Composition of Applications
+
+The following diagram explains the application composition, it is internationally simplified, learn more details on the [Marketplace infrastructure system requirements here](/docs/marketplace/dev/setup/system-infrastructure-requirements.html).
```

Comment:
I'm missing top to bottom composition that explains how MP is played for B2C and B2B shops: where marketplace is an extension to them and also an independent appplication (a case when Magnto is used for B2C as a channel, but Spryker MP is used or MP capabilities over API in a Mirakl-like setup).